news/2021-02-26-agm.md

```diff
@@ -0,0 +1,196 @@
+---
+title: AGM 2021
+---
+
+HackSoc will be holding its Annual General Meeting (AGM) on
+**Friday 5th March at 6pm**!
+
+This year's AGM will be held online. If you have nominated yourself
+for a committee position, join using [this Google Meet
+link](https://meet.google.com/oqe-djht-bmo) so that you can speak.
+
+If you aren't thinking of nominating yourself for a position, then
+please watch the live stream of the meeting instead! This will be
+streamed using [our live streaming service](https://live.hacksoc.org/)
+- a direct link to the stream will be emailed to you when the AGM
```

Comment:
Oops, almost missed this. Seems like the line wrapped on a hyphen here and created a rogue list.

Reply:
Good catch, should be fixed now :)